code/modules/mob/living/carbon/human/species/skrell.dm

```diff
@@ -43,4 +44,12 @@
 		"is jamming their thumbs into their eye sockets!",
 		"is twisting their own neck!",
 		"makes like a fish and suffocates!",
-		"is strangling themselves with their own tendrils!")
\ No newline at end of file
+		"is strangling themselves with their own tendrils!")
+
+/datum/species/skrell/on_species_gain(mob/living/carbon/human/H)
+	..()
+	ADD_TRAIT(H, TRAIT_WATERBREATH, "species")
+
+/datum/species/skrell/on_species_loss(mob/living/carbon/human/H)
+	..()
+	REMOVE_TRAIT(H, TRAIT_WATERBREATH, "species")
```

Comment:
Can't you just add the trait to the `species_traits ` list instead of `ADD_TRAIT` and `REMOVE_TRAIT`?

https://github.com/ParadiseSS13/Paradise/pull/12718/files#diff-6befeb8738a945b5590591fce266433cR16

Reply:
I assume the plan is to eventually get rid of species_trait entirely and just use the framework we have here, better to future proof this PR it’s also more readable to have HAS_TRAIT instead of TRAIT_X in dna.species.species_traits

Reply:
right but then with multiple traits you'd have to spam ADD/REMOVE

Reply:
When all species traits are turned into real ones you can just go through the list and add them in a loop like TG does, I didn’t want to ask lumpy to do this because this is their first PR